app/TuistApp/Sources/Extensions/SimulatorDeviceAndRuntime+Extras.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import TuistCore
+
+extension [SimulatorDeviceAndRuntime] {
```

Comment:
Should we achieve this by making `SimulatorDeviceAndRuntime` conform the [Comparable](https://developer.apple.com/documentation/swift/comparable) protocol?